Untitled-1.js

```diff
@@ -0,0 +1,25 @@
+
+
+
+class Person {
+   constructor(attributes) {
+     this.age = attributes.age;
+     this.name = attributes.name;
+   }
```

Comment:
This bit is a little confusing being in a organized good looking code way lol. I would try putting all of your classes first like for person, instructor, and then project manager so that you can see the parent and what is being extended through each before moving on to the end with the standup time console log